articles/topics/why_intent_architect.md

```diff
@@ -137,7 +137,7 @@ In a similar way that package managers such as NuGet and NPM provide a platform
 Intent's Module system operates in a similar way to these package managers, providing searching, versioning, dependency management, and configuration. With it, individual developers and organizations can leverage the patterns built and maintained by experts.
 
 >[!TIP]
->Learn how to build your own modules by following [this guide](../getting_started/create_your_own_module.md).
+>Learn how to build your own modules by following [this guide](xref:TutorialCreateModule).
```

Comment:
As per one of my earlier comments, we could leave the square brackets empty to have the article name resolved automatically.